3.3/examples/django/application-template-stibuild.json

```diff
@@ -0,0 +1,282 @@
+{
+  "apiVersion": "v1beta1",
+  "items": [
+    {
+      "apiVersion": "v1beta1",
+      "containerPort": 8080,
+      "id": "frontend",
+      "kind": "Service",
+      "port": 5432,
+      "selector": {
+        "name": "frontend"
+      }
+    },
+    {
+      "apiVersion": "v1beta1",
+      "kind": "ImageStream",
+      "metadata": {
+        "name": "django-example"
+      }
+    },
+    {
+      "apiVersion": "v1beta1",
+      "kind": "ImageStream",
+      "metadata": {
+        "name": "python-33-centos7"
+      },
+      "spec": {
+        "dockerImageRepository": "openshift/python-33-centos7"
+      }
+    },
+    {
+      "apiVersion": "v1beta1",
+      "kind": "BuildConfig",
+      "metadata": {
+        "name": "django-example-build",
+        "labels": {
+          "name": "django-example-build"
+       }
+      },
+      "parameters": {
+        "output": {
+          "to": {
+            "name": "django-example"
+          }
+        },
+        "source": {
+          "git": {
+            "uri": "git://github.com/openshift/sti-python.git"
+          },
+          "type": "Git",
+          "contextDir": "3.3/examples/django"
+        },
+        "strategy": {
+          "stiStrategy": {
+            "image": "openshift/python-33-centos7"
+          },
+          "type": "STI"
+        }
+      },
+      "triggers": [
+        {
+          "github": {
+            "secret": "secret101"
+          },
+          "type": "github"
+        },
```

Comment:
You're not using the GH webhook anywhere in the code, I'd remove it from here, so this file is as small as needed.